docs/content/en/docs/clustermgmt/cluster-upgrades/baremetal-upgrades.md

```diff
@@ -84,6 +84,18 @@ eksa-worker2                    type=worker-group-1
 
 If you don't have any available hardware that match this requirement in the cluster, you can [setup a new hardware CSV]({{< relref "../../getting-started/baremetal/bare-preparation/#prepare-hardware-inventory" >}}). You can feed this hardware inventory file during the [upgrade cluster command]({{< relref "baremetal-upgrades/#upgrade-cluster-command" >}}).
 
+#### Skip BMC connectivity checks for faulty machines
+
+EKS Anywhere validates that all BMC machines are contactable before performing cluster upgrades. If you have faulty BMC machines with connectivity issues or hardware faults, you can skip validation for those specific machines so they don't block your cluster upgrade.
+
+To skip BMC validation for a machine:
+
+```
+kubectl label machine.bmc.tinkerbell.org <machine-name> \
```

Comment:
nit: Do we need to provide any guidance on if/when customer should remove the label and how?